### PR TITLE
Sync notebooks with the site.

### PR DESCRIPTION
**Pull request by MarkDaoust**
Fix formatting.
Add licenses.
Fix buttons.

### CODE REVIEW

File: site/en/docs/semantic_retriever.ipynb [hunk 1]
```diff
@@ -0,0 +1,1031 @@
+{
```
Comment (markmcd) [inline site/en/docs/semantic_retriever.ipynb:1]: <p>Are we able to strip these too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (markmcd) [inline site/en/docs/semantic_retriever.ipynb:1]: <p>Looks like a missing quote in the first button <code>&lt;a&gt;</code> tag</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: site/en/tutorials/embeddings_quickstart.ipynb [hunk 2]
```diff
@@ -37,7 +37,7 @@
         "id": "title"
```
Comment (markmcd) [inline site/en/tutorials/embeddings_quickstart.ipynb:37]: <p>This file was deleted at HEAD</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Comment by markmcd** [comment]
Can we get the URL to be `ai.google.dev` for the site buttons? Not all of them will redirect.

File: site/en/examples/chat_calculator.ipynb [hunk 3]
```diff
@@ -48,7 +48,7 @@
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://developers.generativeai.google/examples/chat_calculator\"><img src=\"https://developers.generativeai.google/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on Generative AI</a>\n",
+        "    <a target=\"_blank\" href=\"https://ai.google.dev/examples/chat_calculator.ipynb\"><img src=\"https://developers.generativeai.google/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on Generative AI</a>\n",
```
Comment (markmcd) [inline site/en/examples/chat_calculator.ipynb:51]: Is the site-link meant to include *.ipynb? Will that still download the notebook?

**Review by markmcd — APPROVED** [review]

File: site/en/docs/semantic_retriever.ipynb (hunk 1 above)
Comment (markmcd) [reply site/en/docs/semantic_retriever.ipynb:1]: Still an issue in the latest push

File: site/en/examples/chat_calculator.ipynb (hunk 3 above)
Comment (MarkDaoust) [reply site/en/examples/chat_calculator.ipynb]: Fixed.

**Pull request merged**